modules/sc-mesh-secure-deployment/src/nats/comms_nats_controller.py

```diff
@@ -272,7 +276,7 @@ def __init__(
 
         self.__cbm_certs_path = "/opt/crypto/ecdsa/birth/filebased"
         self.__cbma_ca_cert_path = "/opt/mspki/ecdsa/certificate_chain.crt"
-        self.__cbma_threads: Dict[str, str] = {}
+        self.__cbma_processes: Dict[str, Process] = {}
         self.__cbma_shutdown_event = threading.Event()
```

Comment:
Should MDM agent should use multiprocessing.Event() instead since CBMA is running in different process. Not sure would it then require further changes on CBMA side to signal its threads.

Reply:
I removed it and changed the CBMA interface. This event is still useful inside CBMA (in changes found in another branch to be added in the future) for handling the threads on different error cases :cheese: